### ISSUE TITLE
redsocks stopped working on Fedora kernel 4.15.6-300.fc27.x86_64

### ISSUE THREAD

**Issue by emx0r** [darkk/redsocks#118]
Hello,

after Fedora kernel upgrade to 4.15.6-300.fc27.x86_64, the redsocks no longer proxifies the connections.
The process stays up, from it's log it looks like it's working fine:

1520328561.550727 notice main.c:167 main(...) redsocks started, conn_max=512

But it does not process any connection. I've also noticed I'm unable to kill the process not even using kill -9.
redsocks process stays still up

**Comment by alex-sever-h** [comment]
Yeah, I just noticed that, too. However, kernel 4.15.7 seems to work.
I just updated it from updates-testing repo:
`sudo dnf update --enablerepo updates-testing kernel`

**Comment by emx0r** [comment]
thanks for info

**Comment by anselmolsm** [comment]
I noticed that this week too.
And redsocks is back to normality with kernel 4.15.7 on Arch Linux as well.
